ButterflyMX Demo/Logic/CallsService.swift

```diff
@@ -36,6 +37,22 @@ class CallsService: NSObject {
 
     private var callController = CXCallController()
     
+    func getPushToken() -> Data? {
+        if CallNotificationTypeManager.shared.getCurrentCallNotificationType() == .videoCall {
+            return pushkitToken
+        } else {
+            return pushNotificationToken
+        }
+    }
+    
+    func endCurrentCall() {
+        if CallNotificationTypeManager.shared.getCurrentCallNotificationType() == .videoCall {
+            endCurrentCallKitCall()
```

Comment:
maybe we can call this in the SDK's callback function "callEnded"? So based on `usingCallKit` we could decide to hide uikit or not?

Reply:
callEnded happens after endCurrentCall()

Reply:
oh, ok, so you end call kit call and it's delegate will call `BMXCallKit.shared.endCall()`, right?

Reply:
Yes